apps/web-client/src/routes/orders/suppliers/[id]/+page.ts

```diff
@@ -1 +1,19 @@
+import { getPossibleOrderLinesForSupplier } from "$lib/db/orders/suppliers";
+import type { BookEntry } from "@librocco/db";
+import type { PageLoad } from "./$types";
+export const load: PageLoad = async ({ parent, params }) => {
+	const { ordersDb } = await parent();
+
+	const lines = await getPossibleOrderLinesForSupplier(ordersDb, parseInt(params.id));
+	const isbns = lines.map((book) => book.isbn);
+	const bookData = (await ordersDb.execO(`SELECT * FROM book WHERE isbn IN (${isbns.join(", ")})`)) as BookEntry[];
+
+	const bookDataMap = new Map<string, BookEntry>();
+	bookData.forEach((book) => {
+		bookDataMap.set(book.isbn, book);
+	});
+
+	const supplierOrderLinesWithData = lines.map((line) => ({ ...line, ...bookDataMap.get(line.isbn) }));
+	return { lines, books: supplierOrderLinesWithData, bookDataMap };
```

Comment:
In the view `data.books` is accessed like a map at one point `data.books[isbn]`. I was a little confused if "books" meant book data or order_line data. I would use `orderLines` or `orderLinesWithBookData` to distinguish it from the idea of a book entity